account-kit/signer/src/signer.ts

```diff
@@ -27,19 +27,30 @@ export type AuthParams =
     }
   | ({
       type: "oauth";
-      authProviderId: string;
-      isCustomProvider?: boolean;
       scope?: string;
       claims?: string;
-    } & RedirectConfig)
+    } & OauthProviderConfig &
+      OauthRedirectConfig)
   | { type: "oauthReturn"; bundle: string; orgId: string };
 
-export type OauthMode = "redirect" | "popup";
+export type OauthProviderConfig =
+  | {
+      authProviderId: "auth0";
+      isCustomProvider?: false;
```

Comment:
isn't this always true for `auth0`?

Reply:
No, custom providers are a feature we haven't yet implemented in the dashboard where the developer brings their own OIDC provider that's not known by Alchemy. So for example if Petco wanted to add a "Log in with your Petco account" button.

Reply:
ahh that makes sense, wasn't sure what a `customProvider` was here